float32 my_x
float32 my_y
float32 my_z
float32 my_degreex
float32 my_degreey
float32 my_degreez
float32 my_degreew
